docs/en/latest/plugins/mocking.md

```diff
@@ -0,0 +1,236 @@
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "methods": ["GET"],
+    "uri": "/index.html",
+    "plugins": {
+        "mocking": {
+            "delay": 1,
+            "content_type": "application/json",
+            "response_status": 200,
+            "response_schema": {
+               "properties":{
+                   "field0":{
+                       "example":"abcd",
+                       "type":"string"
+                   },
+                   "field1":{
+                       "example":123.12,
+                       "type":"number"
+                   },
+                   "field3":{
+                       "properties":{
+                           "field3_1":{
+                               "type":"string"
+                           },
+                           "field3_2":{
+                               "properties":{
+                                   "field3_2_1":{
+                                       "example":true,
+                                       "type":"boolean"
+                                   },
+                                   "field3_2_2":{
+                                       "items":{
+                                           "example":155.55,
+                                           "type":"integer"
+                                       },
+                                       "type":"array"
+                                   }
+                               },
+                               "type":"object"
+                           }
+                       },
+                       "type":"object"
+                   },
+                   "field2":{
+                       "items":{
+                           "type":"string"
+                       },
+                       "type":"array"
+                   }
+               },
+               "type":"object"
+           }
+        }
+    },
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "127.0.0.1:1980": 1
+        }
+    }
+}'
+```
+
+## Test Plugin
+
+the `mocking` plugin is configured as follows:
+
+```json
+{
+  "delay":0,
+  "content_type":"",
+  "with_mock_header":true,
+  "response_status":201,
+  "response_example":"{\"a\":1,\"b\":2}"
+}
+```
+
+Use curl to access:
+
+```shell
+$ curl http://127.0.0.1:9080/test-mock -i
+HTTP/1.1 201 Created
+Date: Fri, 14 Jan 2022 11:49:34 GMT
+Content-Type: application/json;charset=utf8
+Transfer-Encoding: chunked
+Connection: keep-alive
+x-mock-by: APISIX/2.10.0
+Server: APISIX/2.10.0
+
+{"a":1,"b":2}
+```
+
+## Disable Plugin
+
+When you want to disable this plugin, it is very simple,
+you can delete the corresponding JSON configuration in the plugin configuration,
+no need to restart the service, it will take effect immediately:
```

Comment:
```suggestion
To disable this plugin, simply delete the corresponding JSON configuration in the plugin configuration,
there is no need to restart the service, the change will take effect immediately:
```
Would be better ?

Reply:
emm, I just copy this from other plugins~